CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 - [server] Restart count on 'app info' output
+- [client] Hanging deploys when the app dir doesn't exist
```

Comment:
This actually **fixes** hanging deploys, right?

Reply:
Yes, typing in a hurry...

Reply:
IMO That's right because that description is already in `fixed` section of the changelog.